packages/pigeon/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-## 20.0.2
+## 20.1.0
 
 * [java] Adds `equals` and `hashCode` support for data classes.
 * [swift] Fully-qualifies types in Equatable extension test.
+* Adds GObject (Linux) support.
```

Comment:
The 20.1.0 and Adds GObject lines should be their own entry, not merged with the previous